keras_nlp/layers/transformer_decoder.py

```diff
@@ -63,6 +63,10 @@ class TransformerDecoder(keras.layers.Layer):
         bias_initializer: string or `keras.initializers` initializer,
             defaults to "zeros". The bias initializer for
             the dense and multiheaded attention layers.
+        normalize_first: bool. If True, the inputs to the attention layer(s) and
+            the intermediate dense layer are normalized (similar to GPT-2). If
+            set to False, outputs of attention layer and intermediate dense
+            layer are normalized (similar to BERT). Defaults to False.
```

Comment:
Move the "Defaults to False." right after "bool.", that will match the other other args in this file.